docs/markdown/snowplow_web_common_cols.md

```diff
@@ -827,4 +827,26 @@ The current timestamp when the model processed this row.
 {% enddocs %}
 
 
+{% docs col_mkt_source_platform %}
+Source platform based off the `utm_source_platform` parameter of the first page_url in the session.
+{% enddocs %}
+
+
+{% docs col_default_channel_group %}
+The channels by which users arrived at your site
```

Comment:
Should we either add an example or otherwise end the descriptions in a full stop? That seems to be our convention